"Fast" setup instructions tweak

The sample URL:
```
git remote add origin https://github.com/unicorn-wg/unicorn-protocol
```
Should probably be ssh/git based:
```
git remote add origin git@github.com:unicorn-wg/unicorn-protocol.git
```

I also find it doesn't work in gh-pages, see https://github.com/richsalz/tls12-frozen/actions/runs/4875095072/jobs/8696865007#step:7:21
```
git -C /tmp/ghpages79 push -qf https://****@github.com/richsalz/tls12-frozen gh-pages
git -C /tmp/ghpages79 push -qf https://****@github.com/richsalz/tls12-frozen gh-pages
make[1]: *** [lib/ghpages.mk:127: ghpages] Error 1
```

--- a/archive.mk
+++ b/archive.mk
@@ -80,7 +80,7 @@ gh-archive: $(ARCHIVE_ROOT)/archive.json
 	  git -C $(ARCHIVE_ROOT) $(CI_AUTHOR) commit -m "Script updating archive at $(shell date -u +%FT%TZ). [ci skip]"; fi
 ifeq (true,$(PUSH_GHPAGES))
 ifneq (,$(if $(CI_HAS_WRITE_KEY),1,$(if $(GITHUB_PUSH_TOKEN),,1)))
-	$(trace) archive -s archive-push git -C $(ARCHIVE_ROOT) push -f https://github.com/$(GITHUB_REPO_FULL) $(ARCHIVE_BRANCH)
+	$(trace) archive -s archive-push git -C $(ARCHIVE_ROOT) push -f "$(shell git remote get-url --push $(GIT_REMOTE))" $(ARCHIVE_BRANCH)
 else
 	@echo git -C $(ARCHIVE_ROOT) push -qf https://github.com/$(GITHUB_REPO_FULL) $(ARCHIVE_BRANCH)
 	@git -C $(ARCHIVE_ROOT) push -qf https://$(GITHUB_PUSH_TOKEN)@github.com/$(GITHUB_REPO_FULL) $(ARCHIVE_BRANCH) >/dev/null 2>&1 \

--- a/ghpages.mk
+++ b/ghpages.mk
@@ -126,7 +126,7 @@ ghpages: cleanup-ghpages $(GHPAGES_ALL)
 	  git -C $(GHPAGES_ROOT) $(CI_AUTHOR) commit -m "Script updating gh-pages from $(shell git rev-parse --short HEAD). [ci skip]"; fi
 ifeq (true,$(PUSH_GHPAGES))
 ifneq (,$(if $(CI_HAS_WRITE_KEY),1,$(if $(GITHUB_PUSH_TOKEN),,1)))
-	$(trace) all -s ghpages-push git -C $(GHPAGES_ROOT) push -f https://github.com/$(GITHUB_REPO_FULL) gh-pages
+	$(trace) all -s ghpages-push git -C $(GHPAGES_ROOT) push -f "$(shell git remote get-url --push $(GIT_REMOTE))" gh-pages
 else
 	@echo git -C $(GHPAGES_ROOT) push -qf https://****@github.com/$(GITHUB_REPO_FULL) gh-pages
 	@git -C $(GHPAGES_ROOT) push -qf https://$(GITHUB_PUSH_TOKEN)@github.com/$(GITHUB_REPO_FULL) gh-pages >/dev/null 2>&1 \

